Week3/Hotel.sql

```diff
@@ -0,0 +1,96 @@
+-- MySQL Workbench Forward Engineering
+
+SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0;
+SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0;
+SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='TRADITIONAL,ALLOW_INVALID_DATES';
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+
+-- -----------------------------------------------------
+-- Schema mydb
+-- -----------------------------------------------------
+CREATE SCHEMA IF NOT EXISTS `mydb` DEFAULT CHARACTER SET utf8 ;
+USE `mydb` ;
+
+-- -----------------------------------------------------
+-- Table `mydb`.`Employees`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`Employees` (
+  `idEmployees` INT NOT NULL,
+  `username` VARCHAR(45) NOT NULL,
+  `Password` VARCHAR(45) NULL,
+  PRIMARY KEY (`idEmployees`),
+  UNIQUE INDEX `username_UNIQUE` (`username` ASC))
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `mydb`.`customer`
+-- -----------------------------------------------------
+CREATE TABLE IF NOT EXISTS `mydb`.`customer` (
+  `idcustomer` INT NOT NULL AUTO_INCREMENT,
+  `first name` VARCHAR(45) NOT NULL,
+  `last name` VARCHAR(45) NOT NULL,
+  `birthday` INT NOT NULL,
+  `customer-num` INT NOT NULL,
+  `Employees_idEmployees` INT NOT NULL,
```

Comment:
This column is hard to understand, which employee should be stored here? 
- Is it the employee that entered this customer's data into the system? A better column name would be: created_by
- Is it the employee who cleans their room? A better column name would be housekeeper_id
- Is it the employee that is responsible for them throughout their stay? 

The name would also help me to understand what the correct relationship would be in the ER diagram.